code/__HELPERS/logging/talk.dm

```diff
@@ -7,7 +7,7 @@
  * Arguments:
  * * message - The message being logged
  * * message_type - the type of log the message is(ATTACK, SAY, etc)
- * * tag - tag that indicates the type of text(announcement, telepathy, etc)
+ * * tag - tag that indicates the type of test(announcement, telepathy, etc)
```

Comment:
Find and replace error?

Reply:
side effect of something intentional to help with the regexing I was doing to cut down on my hand stress. I uh, may have forgotten how to spell